build.gradle

```diff
@@ -52,7 +52,7 @@ allprojects {
         threetenabpVersion        =    '1.1.0'
         universalPkgDir           =    "universal"
         composeVersion            =    '1.2.0-alpha08'
-        composeActivityVersion    =    '1.7.2'
+        composeActivityVersion    =    '1.3.0'
```

Comment:
why composeActivityVersion downgraded ?

Reply:
The one used in the Demo App is 1.3.0. Earlier, the variable composeActivityVersion holding the value 1.7.2 was no where used. In FluentUI.Demo/build.gradle file, I replaced the value '1.3.0' with '$composeActivityVersion' and changed the value of composeActivityVersion from 1.7.2 to 1.3.0